cmd/access/main.go

```diff
@@ -325,8 +329,11 @@ func main() {
 			finalizationDistributor.AddOnBlockFinalizedConsumer(sync.OnFinalizedBlock)
 
 			return sync, nil
-		}).
-		Component("ping engine", func(node *cmd.FlowNodeBuilder) (module.ReadyDoneAware, error) {
+		})
+
+	// the ping engine is only needed for the staked access node
```

Comment:
I'm not sure this isn't needed: cf, my comment on unstaked node fault detection. We might need a one-directional ping engine (from the staked AN to the unstaked AN) though.

Reply:
The ping will be useful but I see that as an additional feature (staked AN being able to track liveness on unstaked AN) and thought I could tackle that separately as a different issue. The current ping reports Node ID and RTT time of all staked nodes and it will have to be changed to report a different metric for unstaked nodes to keep them separate on Grafana.